services/gem/gem-helpers.spec.js

```diff
@@ -0,0 +1,17 @@
+import { test, given } from 'sazerac'
+import { latest, versionColor } from './gem-helpers.js'
+
+describe('Gem helpers', function () {
```

Comment:
I haven't written loads of test cases here. For the most part I think we can rely on the tests for the underlying library in https://github.com/renovatebot/ruby-semver/blob/main/test/index.test.ts I have covered a handful of cases, especially specific to the sorting of pre/stable releases here